models/meshmodel/registry/error.go

```diff
@@ -1,14 +1,39 @@
 package registry
 
 import (
+	"fmt"
+
 	"github.com/layer5io/meshkit/errors"
 )
 
 var (
-	ErrUnknownHostCode    = "meshkit-11146"
-	ErrRegisterEntityCode = ""
+	ErrUnknownHostCode                 = "meshkit-11146"
+	ErrEmptySchemaCode                 = "meshkit-11098"
+	ErrMarshalingRegisteryAttemptsCode = "meshkit-11099"
+	ErrWritingRegisteryAttemptsCode    = "meshkit-11100"
+	ErrRegisteringEntityCode           = "meshkit-11101"
+	ErrUnknownHostInMapCode            = "meshkit-11102"
+	ErrCreatingUserDataDirectoryCode   = "meshkit-11103"
 )
 
 func ErrUnknownHost(err error) error {
 	return errors.New(ErrUnknownHostCode, errors.Alert, []string{"host is not supported"}, []string{err.Error()}, []string{"The component's host is not supported by the version of server you are running"}, []string{"Try upgrading to latest available version"})
 }
+func ErrUnknownHostInMap() error {
+	return errors.New(ErrUnknownHostCode, errors.Alert, []string{"Registrant has no error or it is not supported or unknown."}, nil, []string{"The host registering the entites has no errors with it's entites or is unknown."}, []string{"Validate the registrant name again or check /server/cmd/registery_attempts.json for futher details"})
```

Comment:
"Registrant has no error...." Hmm. What?

Reply:
The part needs rephrasing did it .
The error is resulted when we access the map of the attempts variable with different registrants and it does not contain that registrant so it shouldn't do anything but vihas mentioned that it might panic so the error is for that. I haven't seen this edge case ever happened so I think this is void but still good to have a error if it happens.